packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -570,6 +610,7 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
 /// Usually, the [global] [Offset] is in the coordinate space of the screen
 /// after conversion to logical pixels and the [local] offset is the same
 /// [Offset], but transformed to a local coordinate space.
+@immutable
 class OffsetPair {
```

Comment:
making this a class with `==` seems dangerous, because one of the values is generated from the other using floating point math that may or may not give you precisely the same results each time.